katello-attach-subscription

```diff
@@ -99,24 +135,769 @@ if @yaml.has_key?(:settings) and @yaml[:settings].is_a?(Hash)
   end
 end
 
+
+# check if incompatible options are present
+if @options[:emptyvisor] and @options[:density]
+  puts "FATAL ERROR: Script can't run with --empty-visor and --check-density options enabled. Choose only one."
+  exit 5
+end
+
+# check if -H options is enabled, if true, check the version of Gem as Satellite >= 6.2 doesn't support 
+if @options[:usedhypervsior]
+  api = ApipieBindings::API.new({:uri => @options[:uri], :username => @options[:user], :password => @options[:pass], :api_version => '2', :timeout => @options[:timeout]})
```

Comment:
I think we should make the API connection global, it's re-created too often now.